_api_app/config/app.php

```diff
@@ -45,6 +45,9 @@
 
     'berta_storage_path' => realpath(__DIR__. '/../../storage'),
 
-    'api_prefix' => env('API_PREFIX', '_api')
+    'api_prefix' => env('API_PREFIX', '_api'),
 
+    'image_max_file_size' => 1024 * 3, // 3MB in kilobytes
+    'image_mimetypes' => 'image/png,image/jpeg,image/gif',
```

Comment:
I would recommend using an array here, for better readability and merge it when the string is necessary.